app/mailers/confirmation_mailer.rb

```diff
@@ -0,0 +1,40 @@
+class ConfirmationMailer < MandrillMailer::TemplateMailer
+  include Rails.application.routes.url_helpers
+  include Devise::Controllers::UrlHelpers
+  default from: 'support@freshfoodconnect.org'
+
+  def send_confirmation(users)
+    users = Array.wrap(users) # in case you pass a single user object
+    emails = users.map{|user| {email: user.email, name: "#{user.first} #{user.last}"}}
+
+    mandrill_mail(
+      template: 'Donation Reminder',
+      to: emails,
+      important: true,
+      inline_css: true,
+      recipient_vars: users.map do |user|
+        { user.email =>
+          {
+            "FIRST_NAME" => user.first,
```

Comment:
Currently we don't even have the users give us their first name, does the email template have a fallback generic greeting?